pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "fgpyo"
-version = "1.2.0"
+version = "1.2.1"
```

I think you want a minor bump since we're adding backwards compatible features:

```suggestion
version = "1.3.0"
```

See diff here:

- https://github.com/fulcrumgenomics/fgpyo/compare/1.2.0...main